igor-core/src/main/java/com/netflix/spinnaker/igor/polling/CommonPollingMonitor.java

```diff
@@ -264,6 +277,10 @@ public Long getLastPoll() {
     return lastPoll.get();
   }
 
+  private boolean isSendingEventsEnabled() {
+    return dynamicConfigService.isEnabled("igor.build.sendEvents", true);
```

Comment:
Not sure we need `igor` in here since the fp is going to be scoped to `igor` the application ... should the default be `true` or `false`?

```
igor:
  build:
    sendEvents: true
```

The `igor` in that config snippet would also seem weird in `igor.yml`.

We do have some config under `spinnaker.build` tho?

Reply:
`spinnaker.build.sendEvents` seems a bit weird though... but it would fit with convention - will switch to that.
def want to default to `true`, since that's the behavior today

Reply:
https://github.com/spinnaker/igor/pull/725